bin/docker-push

```diff
@@ -17,7 +17,7 @@ then
   exit 1
 fi
 
-docker tag $DOCKERTAG:latest ${EXTERNAL_REGISTRY_BASE_DOMAIN}/${REPOSITORY_NAME}:$CI_BRANCH
+docker tag -f $DOCKERTAG:latest ${EXTERNAL_REGISTRY_BASE_DOMAIN}/${REPOSITORY_NAME}:$CI_BRANCH
```

Comment:
Makes sense. This checkout is likely the latest version of the branch by time it builds, minus weird concurrency issues.